packages/inertia/src/router.ts

```diff
@@ -77,11 +77,18 @@ export class Router {
   }
 
   protected resetScrollPositions(): void {
-    document.documentElement.scrollTop = 0
-    document.documentElement.scrollLeft = 0
+    const canScrollTo = typeof document.documentElement.scrollTo === 'function'
+    if (canScrollTo) document.documentElement.scrollTo({ top: 0, left: 0 })
+    else {
+      document.documentElement.scrollTop = 0
+      document.documentElement.scrollLeft = 0
+    }
```

Comment:
The window API seems to be full supported, so changing this to window would eliminate the need to check if you can use `scrollTo`, see https://caniuse.com/mdn-api_window_scrollto
The options parameter has less support, so it would probably be best to use `scrollTo(0, 0)`, see https://caniuse.com/mdn-api_window_scrollto_options_top_parameter

```suggestion
    window.scrollTo(0, 0)
```